linera-core/src/node.rs

```diff
@@ -188,6 +188,10 @@ pub enum NodeError {
     #[error("We don't have the value for the certificate.")]
     MissingCertificateValue,
 
+    #[error("Reponse doesn't contain requested ceritifcates: {missing}")]
+    // NOTE: Can't use Vec<CryptoHash> b/c we cannot write Display impl for it.
```

Comment:
Why? We're using `Vec<BlobId>` above, too.

Reply:
It was complaining about not having `AsDisplay` for `Vec<CryptoHash>` and when I tried to implement it then it complained that I cannot write impls for types I don't own.

Reply:
But why is it working for `Vec<BlobId>` then? :thinking:

Reply:
Ah, we're using `0:?` which must be doing some tricks internally (b/c without `:?` it doesn't work).

Reply:
Right, that's it: `?` uses `Debug`, which `Vec<CryptoHash>` _does_ implement.